lib/firestarter/app_builder.rb

```diff
@@ -308,13 +308,19 @@ def setup_metrics
       copy_file "metrics.reek", ".metrics.reek"
     end
 
+    def override_gemfile_lock_for_tests
+      return unless ENV["TESTING"]
```

Comment:
I suggest splitting the issue here. We have the precedent set by `override_path_for_tests` which seems to be wrong. Fixing it in this PR is kinda out of scope, so I would suggest moving it to another PR. What do you think @zamith?